components/sidebar/header/dropdown/sidebar_header_dropdown.jsx

```diff
@@ -486,6 +488,32 @@ export default class SidebarHeaderDropdown extends React.Component {
             );
         }
 
+        let pluginDivider = null;
+        const pluginMenuItems = this.props.pluginMenuItems;
+        const pluginItems = [];
+        if (pluginMenuItems.length > 0) {
+            pluginMenuItems.forEach((item) => {
```

Comment:
nit: This seems like a perfect case for `map`?

Reply:
It would be. It would also make the if statement here redundant as well

Reply:
Yep, good point. Will update it